test/libsolidity/syntaxTests/controlFlow/uninitializedAccess/reverting_call_free_function.sol

```diff
@@ -0,0 +1,13 @@
+
+function iWillRevert() pure { revert(); }
+
+contract C
+{
+        function test(bool _param) pure external returns(uint256)
+        {
+                if (_param) return 1;
+
+                iWillRevert();
+        }
+}
```

Comment:
These test cases do not follow our recommended coding style for Solidity code:

```suggestion
contract C {
    function test(bool _param) pure external returns(uint256) {
        if (_param)
            return 1;

        iWillRevert();
    }
}
```

Reply:
I think it's not so important for tests